.env

```diff
@@ -0,0 +1,16 @@
+# Private AWS configuration parameters to be used locally and for deployment by node-lambda
+AWS_ENVIRONMENT=development
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_PROFILE=
+AWS_SESSION_TOKEN=
+AWS_ROLE_ARN=
```

Comment:
Personally I'd leave off `_ROLE.. _SESSION.., _SECRET.., _ACCESS_KEY..`, and `_ENVIRONMENT` because our practice is to define these in `package.json`